specs/Device/Device/doc/spec.md

```diff
@@ -185,6 +185,19 @@ The data model is defined as shown below:
             -   Type: [DateTime](http://schema.org/DateTime)
     -   Optional
 
+-   `rssi` : Received signal strength indicator for a wireless enabled device.
+    It must be equal to `1.0` when the signal strength is maximum. `0.0` when
+    signal is missing. `-1.0` when it cannot be determined.
+
+    -   Type: [Number](https://schema.org/Number)
+    -   Allowed values: Interval \[-1,0,1\]
```

Comment:
be careful, allowed values a re interval [0,1] and -1 in the exceptional situation that cannot be determined